vgg.py

```diff
@@ -4,31 +4,32 @@
 import numpy as np
 import scipy.io
 
+vgg19_layers = (
```

Comment:
Fixed in https://github.com/anishathalye/neural-style/pull/50/commits/19cdacb6fd90ecf1e95be8897a2f5c3ad87f8847